contrib_bots/lib/github_issues.py

```diff
@@ -19,18 +19,13 @@ class IssueHandler(object):
 
     URL = 'https://api.github.com/repos/{}/{}/issues'
     CHARACTER_LIMIT = 70
-    CONFIG_FILE = '~/.github-issue-bot/github-issue-bot.conf'
+    CONFIG_FILE = '~/.github-auth.conf'
```

Comment:
Makes sense for REPO_NAME and REPO_OWNER, but why should CONFIG_FILE be an instance variable, since it'll point to the same file each time, but the contents of the file could be different; the path shouldn't change.